scripts/kata-install/Dockerfile

```diff
@@ -6,10 +6,9 @@ ADD 50-kata-remote configuration-remote.toml /files/
 
 RUN mkdir -p /scripts
 
-ADD osc-kata-install.sh osc-configs-script.sh osc-log-level.sh lib.sh /scripts/
+ADD osc-kata-install.sh osc-configs-script.sh osc-log-level.sh lib.sh osc-kata-addons-install.sh /scripts/
 
 RUN curl -sSL "https://github.com/opencontainers/umoci/releases/download/v0.4.7/umoci.amd64" -o "/usr/local/bin/umoci" &&\
```

Comment:
@ANJANA-ARK please, help me to understand something...

This is something your PR is not touching, but the amd64 got my attention here... 

umoci v0.4.7 only has amd64 binaries. Is this intended to work only on x86_64 clusters? Or is this work focused on s390x as well? Will this work?